app/server/models/ussd.py

```diff
@@ -49,6 +50,10 @@ def set_data(self, key, value):
             self.session_data = {}
         self.session_data[key] = value
 
+        # https://stackoverflow.com/questions/42559434/updates-to-json-field-dont-persist-to-db
```

Comment:
this took a while to figure out and seems really easy to make the same mistake elsewhere that uses json

Reply:
yeah I've been caught out by this. annoying that it's required.